pyproject.toml

```diff
@@ -47,7 +47,9 @@ geoalchemy = [
 mongoengine = [  # TODO: seems out-of-date/unmaintained; replace or deprecate?
     "Flask-Admin[sqlalchemy]",
     "flask-mongoengine<1",
-    "Flask<2.3.0",  # flask-mongoengine tries to access `flask.json`
+    "Flask<2.3.0",  # flask-mongoengine tries to access `flask.json`,
+    "sqlalchemy>=1.4,<2",
```

Comment:
I don't think we need to repeat the `>=1.4` here, as we get this from the `sqlalchemy` optional group already.

However it's all a bit moot, as I have a PR up to remove support for `mongoengine` altogether.